.travis.yml

```diff
@@ -0,0 +1,12 @@
+sudo: false
+language: node_js
+node_js:
+  - "0.10"
```

Comment:
Does it work? Does it do what we need it to do? Great. Don’t fix what isn’t broken. :)